opm/simulators/wells/WellInterface.hpp

```diff
@@ -317,6 +319,8 @@ namespace Opm
 
         Well well_ecl_;
 
+        const ParallelWellInfo* parallel_well_info_;
```

Comment:
Why a pointer and not a reference?

Reply:
I think we need to be able to copy the wells? Anyhow I will not change this.

Reply:
> I think we need to be able to copy the wells?

That is no reason to avoid using a reference. If it cannot be null and need not be reset it should be a reference.

> Anyhow I will not change this.

Huh?